### ISSUE TITLE
Details about pre-trained 2D_versatile_he model

### ISSUE THREAD

**Issue by Minstrellee** [stardist/stardist#247]
Thank you so much for the support about reproducing experiments.

I got a problem about training the Stardist model on the MoNuSeg dataset. My trained model performed terribly on the MoNuSeg test set, comparing with your pre-trained 2D_versatile_he model.

So I was wondering what's your pre-processing methods on the MoNuSeg dataset. Did you perform normalization or how do you crop the image. What's the val set ratio or the data augmentation method？

I would greatly appreciate it if you could kindly share some insights regarding the training specifics. Thank you very much.

**Comment by titusgriebel** [comment]
I also wanted to ask for specifics on how the 2D_versatile_he model was trained, especially concerning the patch size.
